plan/match_property.go

```diff
@@ -258,6 +258,21 @@ func (p *Projection) matchProperty(_ *requiredProperty, childPlanInfo ...*physic
 }
 
 // matchProperty implements PhysicalPlan matchProperty interface.
+func (p *Analyze) matchProperty(_ *requiredProperty, childPlanInfo ...*physicalPlanInfo) *physicalPlanInfo {
```

Comment:
I used it because I need a way to convert PhysicalPlan `Analyze` to `physicalPlanInfo` by combine all of it's `childPlanInfo`, just like the `Union`.